<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStu</class>
 <widget class="QDialog" name="addStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加学生</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>添加学生信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="name" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入姓名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="num" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_num">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_num">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入学号(不少于10位)</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sex" stretch="3,3,3">
     <item>
      <widget class="QLabel" name="label_sex">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_male">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>男</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_female">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>女</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="age" stretch="1,3,5">
     <item>
      <widget class="QLabel" name="label_age">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_age">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>26</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="major" stretch="1,3,5">
     <item>
      <widget class="QLabel" name="label_major">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>院系：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_major">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>计算机系</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>数字媒体系</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>机械系</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>商务管理学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高职学院</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ins" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_ins">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>兴趣：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="basketball">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>篮球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="football">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>足球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="badminton">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>羽毛球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="volleyball">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>排球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btn" stretch="3,3,3">
     <item>
      <widget class="QPushButton" name="pbtn_OK">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_cancel">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="insGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
